src/Garn/Optparse.hs

```diff
@@ -165,3 +198,6 @@ data CheckCommandOptions
   = Qualified CommandOptions
   | Unqualified
   deriving stock (Eq, Show)
+
+data HiddenParser = Hidden | NotHidden
```

Comment:
`NotHidden` is not used. I think I don't understand why `edit` is showing up as a subcommand. Maybe we should have tests for the behavior we want?

Reply:
The original purpose of this was because: if `edit` is not hidden, instead of `garn COMMAND` in the help text, we get `garn (COMMAND | COMMAND)`. My thought was that if it was not hidden, then *when garn.ts is invalid*, we would get just `garn`, not `garn COMMAND`. But currently the help text doesn't even show up there because the `garn.ts` error takes precedence, so it doesn't really matter.